contentcuration/contentcuration/settings.py

```diff
@@ -89,10 +89,17 @@
 
 SESSION_ENGINE = "django.contrib.sessions.backends.cached_db"
 
+REDIS_URL = "redis://:{password}@{endpoint}:/".format(
+    password=os.getenv("CELERY_REDIS_PASSWORD") or "",
+    endpoint=os.getenv("CELERY_BROKER_ENDPOINT") or "localhost:6379")
+
 CACHES = {
     'default': {
-        'BACKEND': 'django.core.cache.backends.db.DatabaseCache',
-        'LOCATION': 'studio_db_cache',
+        'BACKEND': 'django_redis.cache.RedisCache',
+        'LOCATION': '{url}{db}'.format(url=REDIS_URL, db=1),
```

Comment:
Can you have the db in here as a variable, maybe `CACHE_REDIS_DB`?

```suggestion
        'LOCATION': '{url}{db}'.format(url=REDIS_URL, db=(os.getenv("CACHE_REDIS_DB") or "1"),
```

Reply:
Although that suggestion looks a bit unclean, so best to move it to another variable!

Reply:
Done!